src/pygambit/tests/test_behav.py

```diff
@@ -8,14 +8,18 @@ def setUp(self):
         self.game = gbt.Game.read_game(
             "test_games/mixed_behavior_game.efg"
         )
-        self.profile_double = self.game.mixed_behavior_profile(False)
-        self.profile_rational = self.game.mixed_behavior_profile(True)
+        self.profile_double = self.game.mixed_behavior_profile(None, rational=False)
```

Comment:
The `None` here is unnecessary given that the `rational=` parameter name has been added